src/mscorlib/shared/System/IO/FileStream.Win32.cs

```diff
@@ -3,14 +3,21 @@
 // See the LICENSE file in the project root for more information.
 
 using Microsoft.Win32.SafeHandles;
-using System.Runtime.InteropServices;
 
 namespace System.IO
 {
     public partial class FileStream : Stream
     {
         private SafeFileHandle OpenHandle(FileMode mode, FileShare share, FileOptions options)
         {
+            // CreateFile2 isn't available on Windows 7
+            return Environment.IsWindows8OrAbove
```

Comment:
You may want to copy these changes over to ProjectN enlistment and make sure that they work there - it will verify both the uap flavor and classic flavor.